### PR TITLE
add common models and common params unit test for llm predictor.

### PR DESCRIPTION
**Pull request by aooxin**
### PR types

Others

### PR changes

Others

### Description

添加llm推理中常见模型和常见参数的单测
在A30单卡暂时耗时如下：
11 passed in 1086.90s (0:18:06)

#### 覆盖下面的模型

  | 模型名称
-- | --
1 | ~~Qwen2.5-1.5B-Instruct-tiny-nhl1~~
2 | **Qwen2.5-7B-Instruct-tiny-nhl2**
3 | ~~Qwen2.5-14B-Instruct-tiny-nhl1~~
4 | **Qwen1.5-MoE-A2.7B-Chat-tiny-nhl2**
6 | ~~Qwen2.5-72B-Instruct-tiny-nhl1~~
7 | **Llama-2-7b-chat-tiny-nhl2**
8 | ~~Llama-2-13b-chat-tiny-nhl1~~
9 | ~~Llama-2-70b-chat-tiny-nhl1~~
11 | ~~Meta-Llama-3-8B-Instruct-tiny-nhl1~~
12 | **Meta-Llama-3.1-8B-Instruct-tiny-nhl2**
13 | ~~Meta-Llama-3.1-70B-Instruct-tiny-nhl1~~

Qwen2.5-1.5B-Instruct在参数测试中覆盖
70B和72B的在多卡中测试

#### 覆盖下面的参数
| 参数名称              | 值    |
|-----------------------|-------|
| **batch_size**           | 4     |
| ~~use_flash_attention~~  | True  |
| ~~avx_model~~            | True  |
| **use_fake_parameter**   | True  |
| **inference_model**      | False |
| ~~block_attn~~           | True  |
| **append_attn**          | True  |

#### 覆盖分布式多卡推理场景
  | 模型名称 | 测精度？
-- | -- | --
1 | Qwen2.5-1.5B-Instruct-tiny | True
2 | Qwen2.5-72B-Instruct-tiny-nhl2 | False
3 | Llama-2-70b-chat-tiny-nhl2 | False
4 | Meta-Llama-3.1-70B-Instruct-tiny-nhl2 | False

### CODE REVIEW

File: tests/llm/test_models_params_predictor.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2024 PaddlePaddle Authors. All Rights Reserved.
```
Comment (yuanlehome) [inline tests/llm/test_models_params_predictor.py]: 文件名改成，test_predictor_v1.py 吧

File: tests/llm/test_models_params_predictor.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        ["__internal_testing__/Qwen/Qwen2.5-7B-Instruct-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/Qwen/Qwen2.5-14B-Instruct-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/Qwen/Qwen1.5-MoE-A2.7B-Chat-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/Qwen/Qwen2.5-72B-Instruct-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Llama-2-7b-chat-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Llama-2-13b-chat-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Llama-2-70b-chat-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Meta-Llama-3-8B-Instruct-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Meta-Llama-3.1-8B-Instruct-tiny-nhl1", AutoModelForCausalLM],
+        ["__internal_testing__/meta-llama/Meta-Llama-3.1-70B-Instruct-tiny-nhl1", AutoModelForCausalLM],
```
Comment (yuanlehome) [inline tests/llm/test_models_params_predictor.py]: 把路径中间的那块给删掉，例如"__internal_testing__/Qwen/Qwen2.5-1.5B-Instruct-tiny-nhl1" -> "__internal_testing__/Qwen2.5-1.5B-Instruct-tiny-nhl1"

File: tests/llm/test_models_params_predictor.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+    distance = levenshtein_distance_optimized(a, b)
+    max_length = max(len(a), len(b))
+    return 1 - (distance / max_length)
+
+
+@parameterized_class(
+    ["model_name_or_path", "model_class"],
+    [
+        ["__internal_testing__/Qwen/Qwen2.5-1.5B-Instruct", AutoModelForCausalLM],
```
Comment (yuanlehome) [inline tests/llm/test_models_params_predictor.py]: 这个因为用的是真实模型，直接"__internal_testing__/Qwen/Qwen2.5-1.5B-Instruct" -> "Qwen/Qwen2.5-1.5B-Instruct" 就可以了

File: tests/llm/test_models_params_predictor.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+            self.assertGreaterEqual(full_match / len(self.golden_result), 0.3)
+            self.assertGreaterEqual(partial_match / len(self.golden_result), 0.4)
+        else:
+            self.assertGreaterEqual(full_match / len(self.golden_result), 0.7)
+            self.assertGreaterEqual(partial_match / len(self.golden_result), 0.9)
+
+
+class CommonGpusInferenceTest(TestMultipleGpus, LLMTest):
+    config_path: str = "./tests/fixtures/llm/predictor.yaml"
+    model_name_or_path: str = "__internal_testing__/Qwen/Qwen2.5-1.5B-Instruct"
```
Comment (yuanlehome) [inline tests/llm/test_models_params_predictor.py]: 同上

File: tests/llm/test_models_params_predictor.py (hunk 1 above)
Comment (aooxin) [reply tests/llm/test_models_params_predictor.py]: done

File: tests/llm/test_models_params_predictor.py (hunk 2 above)
Comment (aooxin) [reply tests/llm/test_models_params_predictor.py]: done

File: tests/llm/test_models_params_predictor.py (hunk 3 above)
Comment (aooxin) [reply tests/llm/test_models_params_predictor.py]: done

File: tests/llm/test_models_params_predictor.py (hunk 4 above)
Comment (aooxin) [reply tests/llm/test_models_params_predictor.py]: done

**Pull request closed without merging**